design-patterns/src/app/extensions/FormActionPatterns.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {
+  Alert,
+  Box,
```

Comment:
nit: I noticed this import isn't used. Can you remove the unused imports for each of these .tsx files? I like to make sure our examples are tight.

```suggestion
```